test/BaselineOutput/SingleDebug/Command/CommandTrainingLrWithStats-1-out.txt

```diff
@@ -0,0 +1 @@
+Saving predictor summary
```

Comment:
>summary [](start = 17, length = 7)

with latest Eric changes ([https://github.com/dotnet/machinelearning/pull/1193](https://github.com/dotnet/machinelearning/pull/1193)) you can just use Common folder if files for Debug and Release are same.